lib/mix/tasks/coveralls.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Mix.Tasks.Coveralls.SafeTravis do
+  @moduledoc false
+
+  use Mix.Task
+
+  @preferred_cli_env :test
+  @shortdoc "A safe `coveralls.travis` variant that doesn't crash on failed upload."
+
+  def run(args) do
+    Mix.Tasks.Coveralls.do_run(args, type: "travis")
+  rescue
```

Comment:
Woah did not know you could [use `rescue` in a `def`](https://hexdocs.pm/elixir/Kernel.html#def/2-rescue-catch-after) like this! Super cool, I think I have some refactors to do.